libcontainer/seccomp/seccomp_linux.go

```diff
@@ -166,6 +167,7 @@ func matchCall(filter *libseccomp.ScmpFilter, call *configs.Syscall) error {
 	// Ignore it, don't error out
 	callNum, err := libseccomp.GetSyscallFromName(call.Name)
 	if err != nil {
+		logrus.Warnf("syscall %s is not supported on this kernel\n", call.Name)
```

Comment:
`...supported by this kernel`.

Otherwise LGTM, but I'm not a maintainer here :)